prow/cmd/crier/main.go

```diff
@@ -271,19 +270,13 @@ func main() {
 	}
 
 	if o.gcsWorkers > 0 || o.k8sGCSWorkers > 0 {
```

Comment:
We will need to deprecate and rename this flag as well

Reply:
So I guess that means adding new flags and deprecating:
```go
fs.IntVar(&o.gcsWorkers, "gcs-workers", 0, "Number of GCS report workers (0 means disabled)")
	fs.IntVar(&o.k8sGCSWorkers, "kubernetes-gcs-workers", 0, "Number of Kubernetes-specific GCS report workers (0 means disabled)")
```

Name of the new flags:
* I would suggest `blob-storage-workers` and `kubernetes-blob-storage-workers`
* alternative: `storage-workers` and `kubernetes-storage-workers` 

Deprecation:
* I guess I should just add a deprecation notice to both flags?
* What would be an appropriate time frame: 2-3 months?

Reply:
`[kubernetes-]blob-storage-workers` SGTM. Yeah, approach would be to make the new flags default to the old flags value but emit a warning if the old flags are used and add an a line about that in `prow/Announcements.md` that they will be removed in three months

Reply:
Thx. Done